CommunityToolkitExample/CommunityToolkitExample.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="CommunityToolkit.Mvvm" Version="[8.2.2,9)" />
+    <PackageReference Include="CommunityToolkit.Mvvm" Version="8.3.0" />
```

Comment:
Why you prefer this way? 8.3.0 is bellow 9.

Reply:
I didn't do this.

Reply:
on purpose